commands/tenants

```diff
@@ -27,27 +31,37 @@ create_subtenant() {
   local domain="$1"
   local username="$2"
   local password="$3"
+  local email="$4"
   shift # Remove domain from $@
   shift # Remove username from $@
   shift # Remove password from $@
+  shift # Remove email from $@
+
+  if [[ -z "$company" ]]; then
+    company="SAG"
+  fi
+  if [[ -z "$contactName" ]]; then
+    contactName="IoT Extensions"
+  fi
```

Comment:
- could we make those two really optional? 
-- i.e. if the user doesnt specify them, they will not be used in the commands?
-- please verify if they are required for `c8y tenants create`; if they are, we should make them required for the script as well